recipe/meta.yaml

```diff
@@ -17,7 +17,7 @@ build:
 requirements:
   host:
     - pip
-    - python >=3.6
+    - python
```

Comment:
as it is an "noarch" package, the pinning could stay

Reply:
added pin back